README.md

```diff
@@ -86,7 +86,7 @@ gcloud auth login --no-activate
 
 To run end-to-end tests: `npx cypress run` or `npx cypress open` (interactive mode)
 
-To run unit tests: `npx cypress run-ct` or `npx cypress open-ct` (interactive mode)
+To run unit tests: `npx cypress run --component` or `npx cypress open --component` (interactive mode)
```

Comment:
ah interesting.  Didn't realize that this had changed

Reply:
Yeah, the old command didn't work for me and there was an error message saying to use the new commands. I believe the old ones will soon be retired.